examples/src/systems.cairo

```diff
@@ -77,10 +77,40 @@ mod tests {
         let mut systems = array::ArrayTrait::<felt252>::new();
         systems.append(SpawnSystem::TEST_CLASS_HASH);
         systems.append(MoveSystem::TEST_CLASS_HASH);
+        // routes
+        let mut routes = array::ArrayTrait::<Route>::new();
+        routes.append(
+            RouteTrait::new(
+                'Move'.into(), // target_id
+                'MovesWriter'.into(), // role_id
+                'Moves'.into(), // resource_id
+            )
+        );
+        routes.append(
+            RouteTrait::new(
+                'Move'.into(), // target_id
+                'PositionWriter'.into(), // role_id
+                'Position'.into(), // resource_id
+            )
+        );
+        routes.append(
+            RouteTrait::new(
+                'Spawn'.into(), // target_id
+                'MovesWriter'.into(), // role_id
+                'Moves'.into(), // resource_id
+            )
+        );
+        routes.append(
+            RouteTrait::new(
+                'Spawn'.into(), // target_id
+                'PositionWriter'.into(), // role_id
+                'Position'.into(), // resource_id
+            )
+        );
```

Comment:
Updating the examples for setting the authentication routes when spawning the test world